app/facades/food_facade.rb

```diff
@@ -0,0 +1,16 @@
+class FoodFacade
+  class << self
+    def get_foods(food_name)
+      data = FoodService.call_for_a_food(food_name)
+      foods = data[:foods].map do |food|
+        Food.new(food)
+      end
+      foods.take(10)
+    end
+
+    def get_total_foods(food_name)
+      data = FoodService.call_for_a_food(food_name)
```

Comment:
If you were to add the additional param in your service to get page size you could make a single call vs two calls. An example
```
def self.foods_by_keyword(keyword)
    foods_results = FoodService.foods_by_keyword(keyword)
    x = foods_results[:foods].map do |food_result|
      Food.new(food_result)
    end
    total_hits, foods = [foods_results[:totalHits], x]
  end
```
The last line is creating an array with foods and count